compiler warnings for ws-auth.cc

```
ws-auth.cc: In function ‘void apiZoneCryptokeys(HttpRequest*, HttpResponse*)’:
ws-auth.cc:551:42: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     if (inquireSingleKey && inquireKeyId != value.second.id) {
                             ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
ws-auth.cc:551:42: warning: ‘inquireKeyId’ may be used uninitialized in this function [-Wmaybe-uninitialized]
     if (inquireSingleKey && inquireKeyId != value.second.id) {
                             ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
```

GCC 6.1

--- a/pdns/ws-auth.cc
+++ b/pdns/ws-auth.cc
@@ -530,7 +530,7 @@ static void apiZoneCryptokeys(HttpRequest* req, HttpResponse* resp) {
     throw ApiException("Only GET is implemented");
 
   bool inquireSingleKey = false;
-  int inquireKeyId;
+  unsigned int inquireKeyId = 0;
   if (req->parameters.count("key_id")) {
     inquireSingleKey = true;
     inquireKeyId = std::stoi(req->parameters["key_id"]);

